src/crash.cpp

```diff
@@ -9,7 +9,7 @@
 #include <typeinfo>
 
 #ifdef TILES
-#include <SDL2/SDL.h>
+#include "sdl_wrappers.h"
```

Comment:
Why this again? Do you use some automated tool that tries to push this change? "sdl_wrappers.h" includes more than just "SDL.h" and only that one header is needed here.

Reply:
Can't you change the include path as given to the compiler (the `-I`option) to include the "SDL2" suffix?

Reply:
SDL2 prefix in include path is only necessary for `vcpkg` Visual Studio solution, so I've made changes to affect only it and leave other without prefix.